react/src/lib/Popover/index.js

```diff
@@ -128,11 +152,43 @@ class Popover extends React.Component {
     }
   };
 
+  // Handle keydown for the trigger element
+  handleKeyDown = e => {
+    switch (e.which) {
+      case 13: // ENTER
+        e.preventDefault();
+        e.stopPropagation();
+        e.persist();
+        
+        // Open Popover
+        this.setState(
+          { isHovering: true },
+          () => this.delayedShow(e)
+        );
+        break;
+      case 27: // ESC
+        e.persist();
+        e.stopPropagation();
+
+        // Close Popover
+        this.setState({isHovering: false}, () =>
```

Comment:
Reason you're not calling your new handleHide function here?

Reply:
Forgot about that one, thanks!